docs/tools/mbed_targets.md

```diff
@@ -151,19 +151,23 @@ When you use target inheritance, you may alter the values of `features` using `f
 
 #### `config` and `overrides`
 
-The list of _configs_ provide a way to modify the values of macros in child targets or in a project. Each configuration has a macro name, as well as a default value and an optional help value. For example:
+The list of _configs_ provide a way to modify the values of macros in child targets or in a project. Each configuration has default value, as well as an optional macro name and help text. By default, the macro name will be the name of the config. For example:
 
 ```json
 "config": {
     "clock_src": {
         "help": "Clock source to use, can be XTAL or RC",
         "value": "XTAL",
-        "macro_name": "CLOCK_SRC"
+    },
+    "clock_freq": {
+        "help": "Clock frequency in Mhz",
+        "value": "16",
+        "macro_name": "CLOCK_FREQUENCY_MHZ"
     }
 }
 ```
 
- This case defines the config `clock_src`, for the `CLOCK_SRC` macro, and has a default value of `XTAL`.
+ This case defines the config `clock_src` with the default value of `XTAL` for the macro `CLOCK_SRC`, and the config `clock_freq` with the default value of 16 for the macro `CLOCK_FREQUENCY_MHZ`.
```

Comment:
have you tested it ? Ibelieve the default name for clock_src value will be `MBED_CONF_CLOCK_SRC` ?

Reply:
I didnt realize there was a standardized macro for that

Reply:
would something like the bluetooth advertising name be a better example maybe?